extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -142,18 +200,41 @@ class AmpApesterMedia extends AMP.BaseElement {
    * @return {string}
    * */
   constructUrlFromMedia_(id) {
-    return `${this.rendererBaseUrl_}/interaction/${encodeURIComponent(id)}`;
+    const conditionalChannelId = this.embedOptions_distributionChannelId ?
+      `&channelId=${this.embedOptions_distributionChannelId}` : '';
+    const conditionalInteractionType =
+      `&type=${encodeURIComponent(this.embedOptions_.playlist ? 'playlist' :
+        'editorial')}`;
+    const platform = `&platform=${getPlatform()}`;
+    const cannonicalUrl = document.querySelector('link[rel=\'canonical\']')
```

Comment:
use `Services.documentInfoForDoc(this.element).canonicalUrl` to get the Canonical Url.